lib/cisco_node_utils/bgp.rb

```diff
@@ -360,6 +360,7 @@ def default_enforce_first_as
     # Nvgen as True With optional 'size <size>
     def event_history_cli
       match = config_get('bgp', 'event_history_cli', @get_args)
+      return default_event_history_cli if match == 'size_small'
```

Comment:
This doesn't make any sense to me as a change. 'size_small' == default_event_history_cli, so this line essentially means:

`return default_event_history_cli if match == default_event_history_cli`

which seems really unnecessary. What problems were you seeing without this check?